GoogleDataTransport/GDTCORLibrary/Internal/GDTCORStorageProtocol.h

```diff
@@ -115,16 +126,21 @@ typedef void (^GDTCORStorageBatchBlock)(NSNumber *_Nullable newBatchID,
 - (void)removeLibraryDataForKey:(NSString *)key
                      onComplete:(void (^)(NSError *_Nullable error))onComplete;
 
-/** Calculates and returns the total disk size that this storage consumes.
- *
- * @param onComplete The callback that will be invoked once storage size calculation is complete.
- */
-- (void)storageSizeWithCallback:(void (^)(GDTCORStorageSizeBytes storageSize))onComplete;
-
 @end
 
 // TODO: Consider complete replacing block based API by promise API.
 
+@protocol GDTCORLibraryData <NSSecureCoding, NSObject>
+@end
```

Comment:
Does/how does this type relate to `GDTCORClientMetrics`?

Reply:
Followup, is the `GDTCORClientMetrics` ultimately what we are storing?

Reply:
Good question. Yeah, maybe it makes sense to be less generic here and use more specific API until generic one is actually required. I'll add a TODO to reconsider it.